src/rpc/mining.cpp

```diff
@@ -391,10 +392,11 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
                             }},
                         {RPCResult::Type::OBJ, "coinbaseaux", "data that should be included in the coinbase's scriptSig content",
                         {
+                            {RPCResult::Type::STR_HEX, "str", "values must be in the coinbase (keys may be ignored)"},
                             {RPCResult::Type::ELISION, "", ""},
```

Comment:
Is elision still needed?

Reply:
Why wouldn't it be? They're an undefined number of key/value pairs.

Reply:
Then it should use the proper type `OBJ_DYN`. Elision is used to omit documentation (e.g to avoid redundancy). It shouldn't be used as a workaround to change the structure of the rpc result in the doc


```diff
diff --git a/src/rpc/mining.cpp b/src/rpc/mining.cpp
index 9200ac3bf7..560c46a13c 100644
--- a/src/rpc/mining.cpp
+++ b/src/rpc/mining.cpp
@@ -542,10 +542,9 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
                                         {RPCResult::Type::NUM, "weight", "total transaction weight, as counted for purposes of block limits"},
                                     }},
                             }},
-                        {RPCResult::Type::OBJ, "coinbaseaux", "data that should be included in the coinbase's scriptSig content",
+                        {RPCResult::Type::OBJ_DYN, "coinbaseaux", "data that should be included in the coinbase's scriptSig content",
                         {
                             {RPCResult::Type::STR_HEX, "str", "values must be in the coinbase (keys may be ignored)"},
-                            {RPCResult::Type::ELISION, "", ""},
                         }},
                         {RPCResult::Type::NUM, "coinbasevalue", "maximum allowable input to coinbase transaction, including the generation award and transaction fees (in satoshis)"},
                         {RPCResult::Type::OBJ, "coinbasetxn", "information for coinbase transaction (in same format as elements of 'transactions' above)",

Reply:
I introduced this bug, but since you are touching the lines here, might as well fix it

Reply:
Fixed